### PR TITLE
Handle JsonSyntaxException in OptionManager

### PR DESCRIPTION
**Pull request by agermain**
This PR deals with #149.

### CODE REVIEW

File: cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java [hunk 1]
```diff
@@ -79,8 +79,10 @@ ValidatorParameters getOptionalValidatorParametersOrDefault(CommandLine commandL
                 fixConventionRenaming(jsonElement, "property");
                 Gson gson = new GsonBuilder().create();
                 parameters = gson.fromJson(jsonElement, ValidatorParameters.class);
-            } catch (Exception ignored) {
+            } catch (java.io.IOException ignored) {
                 System.out.println("Invalid path to option files, using default.");
+            } catch(com.google.gson.JsonSyntaxException e) {
```
Comment (JFCote) [inline cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java:84]: The code is good. Could you do a "format" or just manually add a space between `catch` and `(` ? 
Thanks!

**Review by JFCote — CHANGES_REQUESTED** [review]
Just a small change

File: cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java (hunk 1 above)
Comment (agermain) [reply cli/src/main/java/org/openapitools/openapistylevalidator/cli/OptionManager.java]: Nice catch.... 😛

**Review by JFCote — APPROVED** [review]

**Comment by JFCote** [comment]
Thank you very much for your help making openapi-style-validator better :)

**Pull request merged**